modules/virt-custom-kubemacpool-range.adoc

```diff
@@ -0,0 +1,78 @@
+// Module included in the following assemblies:
+//
+// * virt/vm_networking/virt-using-mac-address-pool-for-vms.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-custom-kubemacpool-range_{context}"]
+= Customizing the MAC pool range
+
+[role="_abstract"]
+KubeMacPool works by allocating MAC addresses to VMs from a range. The `rangeStart` and `rangeEnd` parameters in the `HyperConverged` custom resource (CR) define the MAC pool range.
+
+As a cluster administrator, you can configure this range to ensure that MAC addresses for VMs hosted on {VirtProductName} do not conflict with other virtualization solutions on the same network.
+
+.Prerequisites
+
+* You have cluster administrator access on an {product-title} cluster.
+* You have installed the {oc-first}.
+
+.Procedure
+
+. Edit the `HyperConverged` CR by running the following command:
++
+[source,terminal,subs="attributes+"]
+----
+$ oc edit hyperconverged kubevirt-hyperconverged -n {CNVNamespace}
+----
+
+. Update the `HyperConverged` CR to configure the `rangeStart` and `rangeEnd` parameters that define your required MAC address range:
++
+[source,yaml]
+----
+apiVersion: hco.kubevirt.io/v1beta1
+kind: HyperConverged
+metadata:
+  name: kubevirt-hyperconverged
+spec:
+  kubeMacPoolConfiguration:
+    rangeStart: "AA:00:00:00:00:00"
+    rangeEnd: "AA:FF:FF:FF:FF:FF"
+# ...
+----
+
+.Verification
+
+. Run the following command and observe the output:
++
+[source,terminal,subs="attributes+"]
+----
+$ oc get hco kubevirt-hyperconverged -n {CNVNamespace} -o=jsonpath='{.spec.kubeMacPoolConfiguration}'
+----
++
+If you have successfully applied the configuration changes, the output shows the new MAC pool range you have configured:
++
+Example output:
++
+[source,terminal]
+----
+{
+  "rangeStart": "AA:BB:CC:00:00:00",
+  "rangeEnd": "FD:FF:FF:FF:FF:FF"
+}
+----
+
+. Optional. Create a new VM and run the following command to check the MAC address of the VM:
++
+[source,terminal]
+----
+$ oc get vm <vm-name> -n <vm-namespace> -o yaml | grep macAddress
+----
++
+Example output:
++
+[source,yaml]
+----
+macAddress: AA:BB:CC:00:00:00:04
+----
++
+If you have successfully applied the configuration changes, the `macAddress` field in the VM interface starts with the prefix you specified in your `macPoolRange`.
```

Comment:
`macPoolRange` was not referred to earlier in the doc.
I don't even think there is any such field anywhere in the context of KubeMacPool.
@RamLavi Please keep me honest here. Thank you.